modules/data.atmosphere/R/metgapfill.R

```diff
@@ -677,6 +677,6 @@ metgapfill <- function(in.path, in.prefix, outfolder, start_date, end_date, lst
   ## 5b. May need to have an edge case for precipitation
   ## Step 6.Replace gaps with debiased time series (perhaps store statistics of fit somewhere as a measure of uncertainty?)
   ## Step 7. Write to outfolder the new NetCDF file
-
+  
```

Comment:
This seems to be a whole bunch of whitespace changes in an otherwise untouched file. Are they necessary? If not, I'd prefer the changes be removed so it's easier to understand the rest of the PR.

Reply:
I originally thought I had to modify the function, but then found out it wouldn't be suitable for what I wanted to do and had to write a different one.  I thought I had reverted back to the original version, but I can commit a version copy-and-pasted from develop.